extensions/amp-base-carousel/1.0/storybook/Basic.js

```diff
@@ -63,6 +63,40 @@ export const _default = () => {
   );
 };
 
+export const mixedLength = () => {
+  const width = number('width', 440);
+  const height = number('height', 225);
+  const slideCount = number('slide count', 7, {min: 0, max: 99});
+  const colorIncrement = Math.floor(255 / (slideCount + 1));
+  const loop = boolean('loop', true);
+  const snap = boolean('snap', true);
+  const mixedLength = boolean('mixed length', true);
+  const controls = select('show controls', ['auto', 'always', 'never']);
+
+  return (
+    <BaseCarousel
+      controls={controls}
+      mixedLength={mixedLength}
+      loop={loop}
+      snap={snap}
+      style={{width, height}}
+    >
+      {Array.from({length: slideCount}, (x, i) => {
+        const v = colorIncrement * (i + 1);
+        return (
+          <div
+            style={{
+              backgroundColor: `rgb(${v}, 100, 100)`,
+              width: `${Math.floor(Math.random() * 200 + 100)}px`,
```

Comment:
Can we instead do a deterministic formula that feels random, but is not really? Otherwise it's hard to test.

Reply:
Added a few hardcoded "random" presets to Storybook instead, PTAL